### DISCUSSION TITLE
Lazy JSON parsing with loophp/collection - documentation

### DISCUSSION THREAD

**Discussion by drupol** [halaxa/json-machine#55] [General]
Hi,

I discovered your library this morning. I really like it, congrats!

I'm planning to add a documentation page on my own library ([loophp/collection](https://github.com/loophp/collection)), would it be ok for you if I do that?
Do you have a particular example that you would like to provide?

I opened a discussion (https://github.com/loophp/collection/discussions/92) feel free to join the chat!.

**Comment by halaxa** [comment]
Hi, thanks for the info. Nice package. This library is completely free and anyone can build anything on top of it. So feel free to publish anything that can be of help to someone. 😉

**Reply by drupol** [reply]
All good! I'll do that tonight.

Thanks and keep up the good work!

**Reply by drupol** [reply]
Job done! https://github.com/loophp/collection/pull/93
